src/osmf/js/osmf_flash.js

```diff
@@ -1237,11 +1220,9 @@ var swfobject = function() {
           if (attObj[i] != Object.prototype[i]) { // filter out prototype additions from other potential libraries
```

Comment:
We are now using hasOwnProperty in such loops:
for (var i in attObj) {
    if (attObj.hasOwnProperty(i)) {
         .....
    }
}
Please do that.